### DISCUSSION TITLE
Traverse a link in reverse

### DISCUSSION THREAD

**Discussion by rajrahul** [geldata/gel#3509] [Q&A] [answered]
Is it possible to traverse the link in reverse?

type Client {
  required property owner -> str;
  required link license -> License; 
}

How to traverse from a License to the Clients?

**Comment by Sikarii** [comment, marked as answer]
You should be able to use a "backlink", which in turn is a computed.
<https://www.edgedb.com/docs/datamodel/computeds#backlinks>

Something along the lines in your `License` type:
```
type License {
    multi link clients := .<license[is Client];
}
```
